easycql/main.go

```diff
@@ -39,15 +39,15 @@ func generate(fname string) (err error) {
 
 	p := parser.Parser{AllStructs: *allStructs}
 	if err := p.Parse(fname, fInfo.IsDir()); err != nil {
-		return fmt.Errorf("Error parsing %v: %v", fname, err)
+		return fmt.Errorf("error parsing %v: %w", fname, err)
```

Comment:
Using `%w` is not necessary here, as we only print the error to the console and never inspect them programatically. Please see [https://blog.golang.org/go1.13-errors#TOC_3.4.](https://blog.golang.org/go1.13-errors#TOC_3.4.) for a discussion on when to wrap and when not.

Also, while we use Go 1.13 already, there might be potential users on the internet who do not, so we might want to keep compatibility promise of two last Go versions (which is 1.12 and 1.13 currently), similar to what gocql offers.

Could you please change the `%w` back to `%v` and keep only the lowercasing in the commit?

Reply:
> while we use Go 1.13 already, there might be potential users on the internet who do not, so we might want to keep compatibility promise of two last Go versions (which is 1.12 and 1.13 currently)

should we change the language version in `go.mod` to `1.12` too? 

Now it is not possible to build easycql with `go 1.12`:
```
gen/decoder.go:330:40: invalid operation: uint64(1) << gocqlTypeMeta.goType.Bits() (shift count type int, must be unsigned integer)
gen/decoder.go:346:40: invalid operation: uint64(1) << gocqlTypeMeta.goType.Bits() (shift count type int, must be unsigned integer)
note: module requires Go 1.13
```

Reply:
Updated